### ISSUE TITLE
Package Project development,Play with RTSP ,But not work

### ISSUE THREAD

**Issue by beyond6699** [ue4plugins/VlcMedia#34]
In Editor mode is work,But when Package Project ,run the exe , It not work with black texture.How to Package Project ?
unreal 14.3
windows 10

**Comment by beyond6699** [comment]
In log There are two warnings ：
It not use VlcMedia to read rtsp://
`[2017.06.29-12.01.54:238][ 11]LogWmfMedia:Warning: Failed to finish resolve rtsp://192.168.0.178:8554/1: `

and
```
LogPluginManager:Warning: Plugin 'VlcMedia' exists at '../../../vlc_NC/Plugins/VlcMedia/VlcMedia.uplugin' and '../../../vlc_NC/Plugins/VlcMedia/VlcMedia.uplugin' - second location will be ignored
LogModuleManager:Warning: ModuleManager: Module 'XMPP' not found - its StaticallyLinkedModuleInitializers function is null.
```

**Comment by gmpreussner** [comment]
You forgot to set the desired player plug-in on your media source asset. UE4 will pick a suitable player automatically if not set. Since both VlcMedia and WmfMedia support RTSP streams, UE4 chose to play it with WmfMedia in your case. However, WMF may not support the actual data format in the stream, so the resolving of the media source seems to fail (i.e. it can't be opened).

Set VlcMedia on your media source asset as the default player and try again. If you are opening the RTSP stream using a URL string from code, then set the _DesiredPlayerName_ property to "VlcMedia" on your MediaPlayer asset before opening the URL.

**Comment by beyond6699** [comment]
I think I did set the  VideoLAN Media Player (VLCMedia) on my media source asset, I just set Windows player,other platform players set to Auto,  But when package project   still the same problem,In Editor mode everthing is ok.  Did I missing something else?

**Comment by beyond6699** [comment]
I got a new Warning in package game.  
`LogVlcMedia:Warning: Failed to create VLC instance ((null))`

**Comment by gmpreussner** [comment]
Ok, I know what's happening. There was actually a bug that prevented VlcMedia from working in packaged builds. It is fixed for 4.17. You will need these changes: https://github.com/ue4plugins/VlcMedia/commit/cf82a58b1fe433f793b16b9a31a715f7d6a3cab1

**Comment by beyond6699** [comment]
OK，It‘s  work！ Thanks！
